feature/platform/transceiver/zr_laser_bias_current/README.md

```diff
@@ -0,0 +1,99 @@
+# TRANSCEIVER-9: Telemetry: 400ZR TX laser bias current telemetry values streaming. 
+
+## Summary
+
+Validate 400ZR optics modules report accurate laser bias current telemetry
+values.
+
+As per CMIS:
+
+Measured Tx laser bias current is represented as a 16-bit unsigned integer with
+the current defined as the 12 full 16-bit value (0 to 65535) with LSB equal to
+2 uA times the multiplier from Byte 01h:160. For a multiplier of 13 1,
+this yields a total measurement range of 0 to 131 mA.
+
+Accuracy must be better than +/-10% of the manufacturer's nominal value over
+specified 15 operating temperature and voltage.
```

Comment:
can you please clarify this sentence in regards to the "15"? Not sure I followed :)

Reply:
This was just supposed to be "Accuracy must be better than +/-10% of the manufacturer's nominal value over
specified operating temperature and voltage."

Updated :)

Reply:
Ah! Thanks :)